samples/bootloader/pm.yml

```diff
@@ -8,6 +8,7 @@ b0_image:
 
 b0:
   span: [b0_image, provision]
+  ram_size: 0x10000
```

Comment:
This is only here to make it easy to test. To see how it works simply build hello world with secure boot enabled.

Reply:
And will be removed in the proper patch